spyder/plugins/plots/plugin.py

```diff
@@ -47,7 +47,8 @@ def __init__(self, parent):
     def get_settings(self):
```

Comment:
@ccordoba12 do you think it is worth having a method like this one as part of the plugin api?

Reply:
I don't think so.